### DISCUSSION TITLE
Pin specific go version

### DISCUSSION THREAD

**Discussion by DONSA** [renovatebot/renovate#10941] [Request Help] [answered]
Hi, renovate will use the latest version of go according to [documentation](https://docs.renovatebot.com/golang/#go-binary-version).
However, it also states that we can pin the go version that Renovate uses with the one specified in go.mod file.

That doesn't seem to be the case because I have version 1.15 in my go.mod file but renovate image is still using 1.16.
Is this a bug or am I interpreting the documentation wrong?

**Comment by rarkins** [comment, marked as answer]
I think our documentation is outdated/wrong. The go authors recommended we treat the line in go.mod as a minimum version and not exact.

We do in general support override of auto detected versions using the `constraints` object and hopefully they works out of the box already. Eg configure constraints.go=1.15

**Reply by HonkingGoose** [reply]
I'm trying to fix the docs problem, see PR;

- https://github.com/renovatebot/renovate/pull/10942
